calculator.go

```diff
@@ -1,13 +1,111 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"math"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+// Add takes 2+ numbers and returns the result of adding them together.
+func Add(a, b float64, rest ...float64) float64 {
+	result := a + b
+	for _, item := range rest {
+		result += item
+	}
+
+	return result
+}
+
+// Subtract takes 2+ numbers and returns the result of subtracting the second
+// from the first.  For variadic args, evaluation is (a - b) - c ...
+func Subtract(first float64, second float64, rest ...float64) float64 {
```

Comment:
`first` and `second` are a bit long; try `a` and `b` (especially as it's symmetric with `Add`).